CMakeLists.txt

```diff
@@ -19,15 +19,23 @@ include(CTest)
 set(PACKAGE_INSTALL_DESTINATION
   ${CMAKE_INSTALL_DATAROOTDIR}/cmake/${PROJECT_NAME}-${PROJECT_VERSION}
 )
+string(TOLOWER "${PROJECT_NAME}" PROJECT_NAME_LOWER)
+set(PROJECT_INSTALL_DATADIR "${CMAKE_INSTALL_DATAROOTDIR}/${PROJECT_NAME_LOWER}")
+set(PROJECT_INSTALL_MODULESDIR "${PROJECT_INSTALL_DATADIR}/modules")
 
-add_subdirectory(public)
+add_subdirectory(src)
 add_subdirectory(tests)
 
 configure_package_config_file(
   "cmake/${PROJECT_NAME}Config.cmake.in"
   "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
   INSTALL_DESTINATION "${PACKAGE_INSTALL_DESTINATION}"
-  PATH_VARS CMAKE_INSTALL_PREFIX PACKAGE_INSTALL_DESTINATION)
+  PATH_VARS
+  CMAKE_INSTALL_PREFIX
+  PACKAGE_INSTALL_DESTINATION
```

Comment:
I think, `PACKAGE_INSTALL_DESTINATION` isn't needed any more?

Reply:
Oh, could you indent the vaiables relative to the `PATH_VARS`? So that one sees, that these are arguments to `PATH_VARS`?

Reply:
> I think, `PACKAGE_INSTALL_DESTINATION` isn't needed any more?

removed

> Oh, could you indent the vaiables relative to the `PATH_VARS`? So that one sees, that these are arguments to `PATH_VARS`?

put it back into a single line, since it is short enough again.